public/manifest.json

```diff
@@ -28,7 +28,7 @@
     "type": "image/png"
   }
 ],
-  "start_url": "https://nicebreakers.herokuapp.com/",
+  "start_url": "/",
```

Comment:
Was this the problem!?!?

Reply:
No, I used a better library from google to do the generation!

Reply:
Which reminds me...